src/common/Navigation/styled.js

```diff
@@ -1,121 +1,95 @@
 import styled from "styled-components";
+import { ReactComponent as SearchIcon } from "../../icons/Search.svg";
 
 export const NavigationWrapper = styled.div`
-    display: flex;
-    justify-content: space-around;
-    align-items: center;
-    height: 94px;
-    max-width: 100%;
-    background-color: #000;
-    @media (max-width: ${({theme}) => theme.breakpoints.mobile}) and (max-width: ${({theme}) => theme.breakpoints.tabletVertical}) {
-        font-size: 13px;
-        flex-direction: column;
-        height: 142px;
-    }
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  background-color: ${({ theme }) => theme.color.black};
+  @media (max-width: ${({ theme }) => theme.breakpoints.mobile}) {
```

Comment:
one breakpoint is always better that two :D

Reply:
I've added more breakpoints. Now it works and looks much better.